lib/scala/docs-generator/src/main/scala/org/enso/DocsGeneratorMain.scala

```diff
@@ -0,0 +1,124 @@
+package org.enso
+
+import java.io._
+import scala.util.Using
+import scala.io.Source
+import scalatags.Text.{all => HTML}
+import HTML._
+import docsgenerator.DocsGenerator._
+import docsgenerator.TreeOfCommonPrefixes._
+
+/** The Docs Generator script.
+  * Used to create HTML documentation for Enso Standard Library, and also JS
+  * files with react components for the "component reference" website.
+  */
```

Comment:
```suggestion
/** The entry point for the documentation generator.
  *
  * The documentation generator is responsible for creating HTML documentation for the Enso standard library. It also generates JavaScript files containing react components for the reference website.
  */
```
The text "reference website" should link to https://enso.org/docs/reference.